CMakeLists.txt

```diff
@@ -77,95 +61,145 @@ elseif(${CMAKE_BUILD_TARGET} MATCHES "Both")
   set(BUILD_UVC_SHARED TRUE)
   set(BUILD_UVC_STATIC TRUE)
 else()
-  message( FATAL_ERROR "Invalid build type ${CMAKE_BUILD_TARGET}" )
+  message(FATAL_ERROR "Invalid build type ${CMAKE_BUILD_TARGET}" )
 endif()
 
 if(BUILD_UVC_SHARED)
   add_library(uvc SHARED ${SOURCES})
-  set_property(TARGET uvc PROPERTY VERSION ${libuvc_VERSION} )
-  set_property(TARGET uvc PROPERTY SOVERSION ${libuvc_VERSION_MAJOR} )
+  set_target_properties(uvc PROPERTIES
+    VERSION ${libuvc_VERSION}
+    SOVERSION ${libuvc_VERSION_MAJOR}
+    EXPORT_NAME UVCShared
+  )
   list(APPEND UVC_TARGETS uvc)
+  if(NOT LibUVC_STATIC)
+    add_library(LibUVC::UVC ALIAS uvc)
```

Comment:
This one available only if libuvc appended via add_subdirectory() from submodule.